runtime/src/accounts_db.rs

```diff
@@ -6868,7 +6881,7 @@ impl AccountsDb {
         let hash = Hash::from_str(string).unwrap();
         let owner = Pubkey::from_str(string).unwrap();
         let lamports = 100_000_000;
-        let space = 0;
+        let space = self.filler_accounts_config.size;
```

Comment:
@t-nelson Do you think we should set the lamports here based on the data size? If the filler accounts have a huge size, it's possible their rent exempt reserve amount would be greater than 0.1 SOL.

This could probably also be a subsequent PR too.

Reply:
> This could probably also be a subsequent PR too.

No sense adding broken code.  Now seems like the right time.

Reply:
@mrbuzz So you're going to need to add a `Rent` parameter to this function, and then call `Rent::minimum_balance()`[^1] with the filler accounts size. To get the `Rent` parameter, get it from `genesis_config` when `maybe_add_filler_accounts()` is called[^2].

This value will be the new lamports value for each new filler account. So like this:
```rust
let rent_exempt_reserve = rent.minimum_balance(space);
let lamports = 100_000_000 + rent_exempt_reserve;
```

^1: https://github.com/solana-labs/solana/blob/41f2fd7fca08f2a5f238a8fc96e51bd4148266c1/sdk/program/src/rent.rs#L55

^2: https://github.com/solana-labs/solana/blob/25d670a13e5f20507f0020b6e927e98e57063757/runtime/src/serde_snapshot.rs#L574

Reply:
> `let lamports = 100_000_000 + rent_exempt_reserve;`

Is the 0.1 SOL significant?  I figure just the reserve shoud be enough